src/openfermion/transforms/_bravyi_kitaev.py

```diff
@@ -11,15 +11,13 @@
 #   limitations under the License.
 
 """Bravyi-Kitaev transform on fermionic operators."""
-from __future__ import absolute_import
 
 from openfermion.ops import QubitOperator
-from openfermion.transforms._fenwick_tree import FenwickTree
+from openfermion.utils import count_qubits
 
 
 def bravyi_kitaev(operator, n_qubits=None):
     """Apply the Bravyi-Kitaev transform and return qubit operator.
```

Comment:
Especially given all the confusion we've had, please give more description of which BK transform and ideally provide references to both Seeley and Bravyi+Kitaev

Reply:
Done.